src/prototypes/error/error.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
```

Comment:
I converted this from JS to MDX to make it easier for me to work with

Reply:
No worries, though at some point we will probably remove these since they tend to degrade over time (see article listings that are covered by big invisible links for a particularly comical example).